crates/bifrost/src/loglet.rs

```diff
@@ -150,6 +153,17 @@ pub trait Loglet: Send + Sync {
     async fn seal(&self) -> Result<(), OperationError>;
 }
 
+#[derive(Default, Clone, Copy, PartialEq, Eq)]
+pub enum FindTailAttr {
```

Comment:
Can this be `FindTailOpts` or `FindTailOptions`?

Reply:
The reason I have chosen this name over `FindTailOptions` is that it conflicts with another FindTailOptions enum defined internally in replicated_loglet. But maybe we can change the other one instead